src/snowflake/snowpark/modin/pandas/general.py

```diff
@@ -2192,7 +2190,10 @@ def qcut(
 
     if labels is not False:
         # Labels require categorical, not yet supported. Use native pandas conversion here to compute result.
-        return pandas.qcut(x.to_pandas(), q, **kwargs)
+        ErrorMessage.not_implemented(
+            "Snowpark pandas API supports only labels=False, if you need support"
```

Comment:
Can you please update the error message to include method name as well. Something like
```Snowpark pandas qcut method supports only labels=False...```

Reply:
absolutely!